src/olympia/browse/urls.py

```diff
@@ -51,6 +51,10 @@
     url('^complete-themes/(?:(?P<category_name>[^/]+)/)?format:rss$',
         ThemeCategoriesRss(), name='browse.themes.rss'),
 
+    # This won't let you browse any themes but detail page needs the url.
+    url('^static-themes/(?P<category>[^/]+)?$', views.extensions,
+        name='browse.static-themes'),
+
```

Comment:
This will be part of the code burned when frontend grants us salvation, but for now it's useful to be able to check the detail page, even though it's Extension-flavour.